tools/buildgen/extract_metadata_from_bazel_xml.py

```diff
@@ -104,6 +104,8 @@ def _extract_rules_from_bazel_xml(xml_tree):
             ]:
                 if rule_name in result:
                     raise Exception('Rule %s already present' % rule_name)
+                if "@poller=" in rule_name:
```

Comment:
Why are we skipping polling tests here?

Reply:
Cmake choked on these symbols, for example `add_executable(xds_end2end_test@engine=libuv ...`. We had discussed on Monday that CMake probably didn't need to have any tests configured where `uses_polling=True`, we primarily rely on bazel, and this effectively eliminates those tests from Cmake.

Reply:
These tests used to be skipped because extract_metadata_from_bazel_xml.py didn't parse sh_tests. Now that I've replaced some sh_tests with cc_tests, we have to exclude them manually. Otherwise, we'd be adding 5x tests the CMakeLists.txt.